yurt/yurt_core/tests/setup_test.py

```diff
@@ -83,6 +83,23 @@ def test_existing(self, mock_run, mock_rec_file_mod):
         run_calls = mock_run.call_args_list
         assert expected_run_calls == run_calls
 
+    @mock.patch('yurt.yurt_core.setup.recursive_file_modify')
+    @mock.patch('yurt.yurt_core.setup.run')
+    def test_existing_good_name(self, *mocks):
+        mock_run = mocks[0]
+        kwargs = {
+            'git_repo': u'git@github.com:yeti/yeti_fan_page.git'
+        }
+        cli_call = assemble_call_args_list("existing", kwargs)
+        self.runner.invoke(main, cli_call)
+        expected_run_calls = [
+            (('git clone git@github.com:yeti/yeti_fan_page.git',),),
```

Comment:
do these have to be tuples? all of the extra commas are a little much, can we make these nested lists and still have this work?